dashboard/test/ui/features/teacher-homepage.feature

```diff
@@ -0,0 +1,16 @@
+@no_ie
+@no_mobile
+@dashboard_db_access
+@pegasus_db_access
```

Comment:
TODO: See if this test works fine without `@no_ie`, `@dashboard_db_access`, `@pegasus_db_access`.